src/UI/Component/MainControls/Factory.php

```diff
@@ -313,4 +315,36 @@ public function slate() : Slate\Factory;
      * @return  \ILIAS\UI\Component\MainControls\Footer
      */
     public function footer(array $links, string $text = '') : Footer;
+
+
+    /**
+     * ---
+     * description:
+     *   purpose: >
+     *     The Mode Info is a section on a page that informs the user that he is
+     *     in a certain mode (e.g. in the preview as a member of a course).
+     *
+     *   composition: >
+     *     The Mode Info MUST contain a title explaining the mode.
+     *     The Mode Info MUST contain a Close Button to leave the
+     *     mode.
+     *   effect: >
+     *      By clicking the Close Button, the user leaves the current
+     *      (application wide) mode.
+     *
+     * context:
+     *   - The Mode Info is used with the Standard Page.
+     *
+     * rules:
+     *   usage:
+     *     1: The Mode Info is unique for the page - there MUST be not more than one.
+     *   interaction:
+     *     1: The Mode Info MUST allow the user to leave the mode.
+     *   accessibility:
+     *     1: The Mode Info informs about an important circumstance, which must be
+     *     recognizable in particular also for persons with a handicap. Therefore a
+     *     role="alert" MUST be implemented.
```

Comment:
We remove the last sentence in 'accessibility' to keep the discussion open which way is the best to make this UI component accessible.